{{cookiecutter.project_name}}/databricks-config/staging/training-job.tf

```diff
@@ -16,7 +16,6 @@ resource "databricks_job" "model_training_job" {
     {% if cookiecutter.include_feature_store == "yes" %}notebook_task {
       notebook_path = "notebooks/TrainWithFeatureStore"
       base_parameters = {
-        env                = local.env
         training_data_path = "/databricks-datasets/nyctaxi-with-zipcodes/subsampled"
         experiment_name    = databricks_mlflow_experiment.experiment.name
         model_name         = "${local.env_prefix}{{cookiecutter.model_name}}"
```

Comment:
Nitpick and note for future consideration: the environment value is used implicitly in this notebook. It's great that we are cleaning up the explicit unused param env, we should audit and figure out how to make "env" naming consistent across these notebooks in the future. For instance, models have environment prefixed in their name, but feature store tables don't.